test/cppsuite/configs/base_test_stress.txt

```diff
@@ -12,50 +13,56 @@ runtime_monitor=
     (
         enabled=false,
         limit=100
+    ),
+    stat_db_size=(
```

Comment:
Should put the bracket on a newline like the other options.

Reply:
862a4c4